cabal.project

```diff
@@ -128,3 +128,9 @@ source-repository-package
   location: https://github.com/input-output-hk/cardano-ledger
   tag: a773ba71eb9a6fb91dda2cdf6c41b2280f2acf7d
   subdir: crypto/test
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-crypto/
+  tag: 4590efa638397e952a51a8994b5543e4ea3c1ecd
+  subdir: .
```

Comment:
Oops, my bad, I removed that.

But why didn't CI warn us about this?

Reply:
Does CI use cabal to build?